src/sentry/api/endpoints/group_hashes_split.py

```diff
@@ -381,8 +383,10 @@ def _render_trees(group: Group, user):
     )
 
     rv = []
-
-    for row in snuba.raw_snql_query(query, referrer="api.group_split.render_grouping_tree")["data"]:
+    request = Request(dataset="events", app_id="grouping", query=query)
```

Comment:
Nit: Do you see a need for being able to find out all the `app_id` being used in Sentry? If yes, it might be worth defining them in a common place and reusing them.

Reply:
Yeah, that is an open question. My original plan was to have these all defined somewhere, like in a config file. However there is a competing school of thought that says these should be auto-populated. I am going to leave them like this for now, since it will let us get a little bit of data to look at while we iterate on what exactly an app_id is and where it is stored.